Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -435,6 +435,15 @@ class MiscellaneousTestCase: XCTestCase {
         }
     }
 
+    func testCanDebugUsinglldb() throws {
```

Comment:
Silly camel case FYI nit: `testCanDebugUsingLLDB` (acronyms should all upcase together).